src/handler.js

```diff
@@ -9,6 +9,7 @@ const DEFAULT_STATE = {
 	_initialized:   true,
 	archives:       {},
 	archivePages:   {},
+	archivesByPage: {},
```

Comment:
Linting errors

Reply:
The whole project is failing linting right now; fixing all of them in the [`update-for-coding-standards` branch](https://github.com/humanmade/repress/tree/update-for-coding-standards) off this one.